src/environments/array.js

```diff
@@ -150,6 +169,10 @@ function parseArray(
         }
     }
 
+    if (singleRow && body.length > 1) {
+        throw new ParseError("Misplaced \\cr.", parser.nextToken);
```

Comment:
Could you move this after the line 157 (`if (next === "\\cr")`)?

Reply:
Done.